assets/js/blocks/cart-checkout/cart/example.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { DEFAULT_HEIGHT } from '@woocommerce/block-settings';
+import { previewProducts } from '@woocommerce/resource-previews';
+
+export const example = {
```

Comment:
This block won’t have this many attributes, I think we can skip a separate `example.js` file and just keep the `example` in the `index.js` file